test/server_test.py

```diff
@@ -235,6 +254,23 @@ def started_imageio(tmpdir, drop_privileges="true"):
         proc.wait()
 
 
+@requires_root
+def test_load_ticket(tmpdir, tmp_dirs, tmp_image, ticket):
+    data = b"a" * 512 + b"b" * 512
+    tmp_image.write(data)
+    with started_imageio(tmpdir, drop_privileges="false", ticket=ticket):
+        conf_dir = tmpdir.join("conf")
+        cfg = config.load(str(conf_dir.join("conf.d", "daemon.conf")))
+        with http.ControlClient(cfg) as c:
+            res = c.get('/tickets/file')
+            assert res.status == http_client.OK
+
+        with http.LocalClient(cfg) as c:
+            res = c.get('/images/file')
+            assert res.read() == data
+            assert res.status == http_client.OK
```

Comment:
This makes sense, but why do need to use a local client instead of remote client? I guess this is the reason you tried to use a non-abstract unix socket for the configuration.

Reply:
I get connection refused with remote client, currently investigating why.

Reply:
Other tests mostly use the configuration contained in the Server instance, that has the obtained port updated. Since here we do not have access to the server instance, I had to set a specific port. But I used `testutil.random_tcp_port()` to obtain an unused one.